src/cm_adapter.ts

```diff
@@ -126,6 +126,11 @@ function runHistoryCommand(cm: CodeMirror, revert: boolean) {
   }
 }
 
+var keys: any = {};
+"Left|Right|Up|Down|Backspace|Delete".split("|").forEach(key => {
+  keys[key] = (cm:CodeMirror) => runScopeHandlers(cm.cm6, {key: key}, "editor");
+});
+
```

Comment:
any chance we can type this better? maybe `Record<string, (cm: CodeMirror) => void>`?